zpush/Dockerfile

```diff
@@ -34,7 +34,9 @@ VOLUME /var/lib/z-push/
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 # install Z-Push
-RUN \
+# hadolint currently does not understand the extended buildkit syntax https://github.com/hadolint/hadolint/issues/347
```

Comment:
I don't think this should be necessary. Did you rebuild the whole stack during your tests? The build time secrets should be obsolete when using the kopano_php image.

Reply:
I just tried on a non-whitelisted machine and now see the error as well:

```
Failed to fetch https://download.kopano.io/supported/core:/8.7/Debian_10/./Packages  401  Unauthorized
```

Basically simply because the repo is still in there the step with the build time secrets needs to be kept indeed.